index.js

```diff
@@ -6,20 +6,50 @@ import { quickSort } from './utils/quickSort.js';
 import { menuConfig } from './dataStore/menuConfig.js';
 
 const tableContainer = document.getElementById('table');
+let countriesLenght = Object.keys(countries).length;
+const TOTAL_AMOUNT = countriesLenght;
 
+const callbacksObject = {
+  sortCallback: (dataKey, isAsc) => {
+    const isDataKey = Boolean(dataKey);
+    if (!isDataKey) {
+      return countriesTable.render(countries, TOTAL_AMOUNT);
+    }
+    isAsc
+      ? countriesTable.render(quickSort(countries, dataKey), TOTAL_AMOUNT)
+      : countriesTable.render(
+          quickSort(countries, dataKey).reverse(),
+          TOTAL_AMOUNT
+        );
```

Comment:
Let's refactor this part
Get your sorted dataset and set to the variable, use operator `?` for this
and then just call `.render()` method with the total amount and needed dataset